bayeux/_src/bayeux.py

```diff
@@ -151,3 +152,68 @@ def transform_fn(x):
         transform_fn=transform_fn,
         initial_state=initial_state)
 
+  @classmethod
+  def from_pymc(cls, pm_model, initial_state=None):
+    import pymc as pm  # pylint: disable=g-import-not-at-top
+    import pymc.sampling.jax as pm_jax  # pylint: disable=g-import-not-at-top
+
+    uc_model = pm.model.transform.conditioning.remove_value_transforms(pm_model)
+    uc_vars = uc_model.value_vars
+    uc_names = [v.name for v in uc_vars]
+
+    logp = pm_jax.get_jaxified_logp(model=uc_model)
+
+    def identity(x):
+      return x
+
+    def none(_):
+      return 0.
+
+    # We have different ideas of forward and backward!
+    fwd_transforms = {
+        k.name: identity if v is None else v.backward
+        for k, v in pm_model.rvs_to_transforms.items()}
+    bwd_transforms = {
+        k.name: identity if v is None else v.forward
+        for k, v in pm_model.rvs_to_transforms.items()}
+
+    def forward_transform(pt):
+      return [fwd_transforms[k](v) for k, v in zip(uc_names, pt)]
+
+    def backward_transform(pt):
+      return [bwd_transforms[k](v) for k, v in zip(uc_names, pt)]
```

Comment:
This won't work for parametrized transforms like `Interval` that require the extra inputs (lower, upper). The way we usually do it is `transform.forward(value, *rv.owner.inputs)`, and later `model.replace_rvs_by_values` so that it uses the values of other RVs and not the RVs themselves